database.js

```diff
@@ -39,6 +41,15 @@ class Database {
     info(`Dropping database: '${this.databasename}'...`);
     return execa('./bin/e2e_teardown_db', [this.databasename], { cwd: this.djangopath });
   }
+
+  async load(dataset) {
+    info(`Loading fixtures for ${dataset}`);
+    return managePy(
+      this.djangopath,
+      ['load_e2e_data', '--datasets', dataset],
+      { env: settings, stdout: 'inherit' },
```

Comment:
settings are asynchronous. So you're passing a promise here.
Use `const { what, ever, setting, you, need } = await settings`

Reply:
Resolved as discussed.